### PR TITLE
2370: Fix date formatting

### PR DESCRIPTION
**Pull request by LeandraH**
### Short description

Uses a date format that Luxon can read for the opening of the iOS calendar app.

### Side effects

Hopefully none.

### Resolved issues

Fixes: #2370 

---

### CODE REVIEW

File: release-notes/unreleased/2370-fix-ios-crash-calendar-export.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: 2370
+show_in_stores: true
+platforms:
+  - ios
+en: Exporting an event to the calendar app works again
+de: Der Eventexport funktioniert wieder
```
Comment (steffenkleinle) [inline release-notes/unreleased/2370-fix-ios-crash-calendar-export.yml:6]: I think this never did not work, did it? I think only viewing the event in the calendar didn't.

**Review by steffenkleinle — APPROVED** [review]
Not tested but LGTM

File: release-notes/unreleased/2370-fix-ios-crash-calendar-export.yml (hunk 1 above)
Comment (LeandraH) [reply release-notes/unreleased/2370-fix-ios-crash-calendar-export.yml:6]: True, ooops

**Review by sarahsporck — APPROVED** [review]
Tested on ios simulator

**Pull request merged**